src/browser/reducers/browserReducer.js

```diff
@@ -0,0 +1,24 @@
+import { QUERY } from '../actions/browserActions';
+import parseURL from '../util/parseURL';
+
+const initialState = {
+  target: 'nos://nos.neo'
+};
+
+function parse(query) {
```

Comment:
Could we use arrow function here? For consistency reasons

Reply:
I've generally been using `function` at the top level unless the arrow function simplifies the code somehow.  One of the benefits is that functions aren't "anonymous" in stack traces, they actually have a name.